open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -165,8 +165,60 @@ public void getOriginalTypes()
          * Calls for new and changed types go here
          */
         update0015LinkedMediaTypes();
+
+        add0043ProcessingStateClassification();
```

Comment:
Add to model 0042 since this change is not big enough to warrant a whole new web page.  

I can not see the corresponding PR for the Egeria docs website that explains how to use this new classification